sympy/printing/mathml.py

```diff
@@ -584,14 +584,19 @@ def _print_Add(self, expr, order=None):
 
     def _print_MatrixBase(self, m):
         brac = self.dom.createElement('mfenced')
+        if self._settings["mat_delim"] == "[":
+            brac.setAttribute('open', '[')
+            brac.setAttribute('close', ']')
         table = self.dom.createElement('mtable')
         for i in range(m.rows):
             x = self.dom.createElement('mtr')
             for j in range(m.cols):
                 y = self.dom.createElement('mtd')
                 y.appendChild(self._print(m[i, j]))
                 x.appendChild(y)
-            table.appendChild(x)
+                table.appendChild(x)
```

Comment:
Is this on the correct tab level? How did the result look like before and after this change?

I'm not sure exactly what the consequences are more than it should be rendering in completely different ways.

Reply:
they both showing same output.